docs/scenario_editor/trigger_boxes.md

```diff
@@ -0,0 +1,71 @@
+# Vehicle Behavior Trigger Boxes
+
+Vehicle Behavior Trigger Boxes define specific actions or behaviors to happen 
+once that trigger box interacts with another vehicle or intersection in the scene. 
+The action can have an effect on the vehicle with the trigger box or the vehicle it is interacting 
+within the scene. 
+
+<div class="img_container">
+    <img class='md_img' src="../imgs/trigger_scene1.png"/>
+    <div class="space"></div>
+    <img class='md_img' src="../imgs/trigger_scene2.png"/>
+</div>
+
+
+## Adding a Trigger Box
+
+1. In the vehicle's "Details" window, click the "Add Component" button at the top and search for "Vehicle Behavior Trigger."
```

Comment:
I think we should add something like:
"Click the vehicle and then go the "Details" window"

If this the first time user does this, can be a bit strange not to know where is the details window.